rom/laghos.cpp

```diff
@@ -1840,25 +1855,43 @@ int main(int argc, char *argv[])
     bool errorIndicatorComputed = false;
     int errorIndicatorVecSize = 0;
 
-    if (rom_online)
+    // If using the greedy algorithm, calculate the error indicator
+    if (rom_build_database && !rom_calc_rel_error)
     {
-        // If using the greedy algorithm, calculate the error indicator
-        if (rom_build_database && !rom_calc_rel_error)
+        if (rom_online && !converged)
```

Comment:
The variable `converged` was introduced in PR 136, but I think it should be given a more specific name in this PR, since it pertains to the `rom_build_database` case only. The current name could be misunderstood as something about the simulation. Maybe a name like `database_converged` or `greedy_converged` would be more descriptive.